src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePack.csproj

```diff
@@ -0,0 +1,22 @@
+<Project>
+  <!-- Import the MSB4U generated common project as we will rely on it. -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(MSBuildForUnity.Common.props))" Condition="Exists('$([MSBuild]::GetPathOfFileAbove(MSBuildForUnity.Common.props))')" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net471</TargetFrameworks>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <BaseIntermediateOutputPath>.obj\</BaseIntermediateOutputPath>
+    <OutputPath>.bin\</OutputPath>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.Build.NoTargets" Version="2.0.1" />
```

Comment:
Rather than specify `Version` here, can we move it up to the `global.json` file?

Reply:
eaa2be235fd219b0221d09230184186387c9749e